apps/web/app/api/v1/management/people/route.ts

```diff
@@ -0,0 +1,42 @@
+import { responses } from "@/lib/api/response";
+import { DatabaseError } from "@formbricks/types/v1/errors";
+import { authenticateRequest } from "@/app/api/v1/auth";
+import { getPeople } from "@formbricks/lib/services/person";
+import { TPerson } from "@formbricks/types/v1/people";
+import { NextResponse } from "next/server";
+import { createPerson } from "@formbricks/lib/services/person";
+
+export async function GET(request: Request) {
+  try {
+    const authentication = await authenticateRequest(request);
+    if (!authentication) {
+      return responses.notAuthenticatedResponse();
+    }
+    const peoples: TPerson[] = await getPeople(authentication.environmentId!);
```

Comment:
`peoples` -> `people`

Reply:
If we are returning an array than would it be better to name it peoples ? 😊

Reply:
Currently we have this notation:
Singular: person
Plural: people
;-)